ignite/templates/app/stargate/app/app.go.plush

```diff
@@ -253,7 +258,7 @@ func New(
 	bApp.SetInterfaceRegistry(interfaceRegistry)
 
 	keys := sdk.NewKVStoreKeys(
-		authtypes.StoreKey, banktypes.StoreKey, stakingtypes.StoreKey,
+		authtypes.StoreKey, authz.ModuleName, banktypes.StoreKey, stakingtypes.StoreKey,
```

Comment:
@ilgooz there's no StoreKey for the authz module but it's a string to identify the module and some of the other modules resolve to the ModuleName so I used that instead.